### DISCUSSION TITLE
Top/right/bottom/left variant for `border-[...]-none`

### DISCUSSION THREAD

**Discussion by DamianGlowala** [tailwindlabs/tailwindcss#12387] [Ideas]
Would it make sense to add top, right, bottom and left variants for `border-[...]-none`? Currently, we can only use `border-none`.

**Comment by andymccray** [comment]
You can set the width to 0 to turn off the border for one side. E.g. `border-r-0` (or `border-x-0` as a shorthand for left/right).
